core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -931,7 +938,9 @@ public VirtualHostBuilder accessLogWriter(AccessLogWriter accessLogWriter, boole
     public VirtualHostBuilder blockingTaskExecutor(ScheduledExecutorService blockingTaskExecutor,
                                                    boolean shutdownOnStop) {
         this.blockingTaskExecutor = requireNonNull(blockingTaskExecutor, "blockingTaskExecutor");
-        shutdownBlockingTaskExecutorOnStop = shutdownOnStop;
+        if (shutdownOnStop) {
```

Comment:
That's a good point. 👍 
I have actually thought about it and tried to throw an exception if the second one is set.
However, I also thought shutting down the first one is valid:
```java
ServerBuilder sb = ...
sb.blockingTaskExecutor(blockingTaskExecutor1, true);
sb.blockingTaskExecutor(blockingTaskExecutor2, true);
```
Even though `blockingTaskExecutor1` is not used and replaced, I think it's still valid to shut down it because the user specify it so. What do you think about it?

Reply:
Makes sense 👍